frontend/src/components/ChatBox/ChatBoxFeed.tsx

```diff
@@ -10,16 +9,21 @@ import './ChatBoxFeed.css';
 
 function ChatBoxFeed({ messages }: { messages: ChatMessage[] }) {
 	const chatboxFeedContainer = useRef<HTMLDivElement>(null);
-	const isOverflow = useIsOverflow(chatboxFeedContainer);
+
+	useEffect(() => {
+		if (chatboxFeedContainer.current) {
+			chatboxFeedContainer.current.scrollTop =
+				chatboxFeedContainer.current.scrollHeight;
+		}
+	}, [messages]);
 
 	return (
-		<div
+		<section
 			className="chat-box-feed"
 			ref={chatboxFeedContainer}
-			// eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
-			tabIndex={isOverflow ? 0 : undefined}
+			aria-live="polite"
```

Comment:
I'm amused by this.
I'm guessing it means that we don't want updates to the feet to interrupt whatever's being said?

Reply:
"polite" is the recommended setting. And no, there is no "rude" setting, I checked ;)
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions